packet/src/ipv4.rs

```diff
@@ -104,6 +104,19 @@ impl Ipv4Hdr {
         hdr.ipv4_ttl = ttl as u8;
         Ipv4Hdr::update_checksum(pkt);
     }
+
+    pub fn set_identification(pkt: &mut Packet, id: u16) {
```

Comment:
Neither of these seem to be used anywhere.

Reply:
Yep. I was initially setting these into the packet to copy over some mcast info. But, I'll remove this.